journal/updated/09-chapter-8.adoc

```diff
@@ -299,58 +304,58 @@ Several strangers were there, and I had satisfaction in being with them,
+but being still in a very weak state, the trial was great.
 On the 15th of sixth month, I left home, accompanied by my son Joseph,
-reached Coventry that evening, sat three meetings on first-day,
+and reached Coventry that evening. Attended three meetings on first-day,
 and the Quarterly Meeting on second.
 
 Went to Warwick on third-day, was at their week-day meeting on fourth,
-and at Eddington meeting on sixth-day, at Shipston meeting on first-day,
-and to Banbury that evening; was at three meetings on secondday,
-and the Quarterly Meeting on third; went same day to Buckingham,
+and at Eddington meeting on sixth-day. At Shipston meeting on first-day,
+and to Banbury that evening; was at three meetings on second-day,
+and the Quarterly Meeting on third. I went the same day to Buckingham,
 was at meeting in the evening, and at the Quarterly Meeting on fourth-day,
 which was large and satisfactory.
-From there to Northampton,
-on fifth-day was at meeting in the evening and at the Quarterly Meeting on sixth-day;
+From there to Northampton;
+on fifth-day was at meeting in the evening, and at the Quarterly Meeting on sixth-day;
 from there to Wellington that evening,
-and lodged at our very kind and honest friend Benjamin Middleton`'s;
-was at Finedon on first-day,
-at their Monthly Meeting on second and on third-day to Ridlington Park;
-from there to Oakham, at their meeting on fifth-day, and on sixth,
+and lodged at our very kind and honest friend Benjamin Middleton`'s.
+Was at Finedon on first-day,
+at their Monthly Meeting on second, and on third-day to Ridlington Park.
+From there to Oakham, at their meeting on fifth-day, and on sixth,
 after having a solemn opportunity with our dear friends there, returned to the Park,
-and on seventh-day, in a renewed sense of Divine love we took leave.
+and on seventh-day, in a renewed sense of Divine love, we took leave.
 On first-day was at their meetings at Leicester; from there to Groby Lodge;
 on fourth-day returned to Leicester,
-it having rested on my mind to be there the first-day following:
-was taken very unwell at meeting; lodged at John Priestman`'s,
-whose wife took great care of me; and getting better,
+it having rested on my mind to be there the first-day following.
+I became very ill at meeting, and lodged at John Priestman`'s,
+whose wife took great care of me. And getting better,
 I was able to sit meetings on first-day,
 and was well satisfied that I did right in staying.
 On second-day I reached home in safety, with the richest reward,
 for which I beg to be truly thankful,
-and that I may be carried safe through this life of trouble into everlasting rest.
+and desire that I may be carried safe through this life of trouble into everlasting rest.
 
-I have thus had one more opportunity, though unexpectedly, to renew my acquaintance,
+I have thus had one more opportunity, though unexpectedly, to renew my acquaintance
```

Comment:
I THINK she's talking about a bunch of people here too... and not just one, so we should change it to acquaintanceS.